### PR TITLE
Add option to playback animation in realtime

### PR DESCRIPTION
**Pull request by Aa20475**
This is a PR for the issue #1215 .

I'm not sure if this is what @morevnaproject meant there. Can someone see this and let me know?

TODO: Still need to give a command line parameter for choosing.

### CODE REVIEW

**Comment by morevnaproject** [comment]
Thank you!

The code looks good, but I suggest to replace option to something more informative. Also, we have all options generally starting with "--".

So, I suggest to replace "nr" to "--benchmark". ^__^
And add explanation in help, something like this:
```
--benchmark - Ignore real-time synchronization and render every frame (used for benchmarks).
```

File: synfig-studio/src/player/main.cpp [hunk 1]
```diff
@@ -83,6 +84,8 @@ int main(int argc, char **argv)
 		std::cout << std::endl;
 		std::cout << "usage: " << std::endl;
 		std::cout << "  " << commandname << " <file.sif|file.sifz> <renderer>" << std::endl;
```
Comment (morevnaproject) [inline synfig-studio/src/player/main.cpp:86]: Please replace " <file.sif|file.sifz> <renderer>" to " <file.sif|file.sifz> <renderer> [--benchmark]"

**Comment by morevnaproject** [comment]
In all other aspects it works perfectly. Waiting for last changes. ^__^

**Comment by morevnaproject** [comment]
@Aa20475 It looks like last commit (Identation fixes) reverts changes in previous commit (Change to user help.)? ^__^

**Comment by morevnaproject** [comment]
Ah, great! Now everything is alright! Can you please rebase your branch on top of the latest master? ^__^

**Comment by ice0** [comment]
@morevnaproject I think is better to "Squash and merge" - also it automatically rebased.

**Comment by Aa20475** [comment]
`@Aa20475 It looks like last commit (Identation fixes) reverts changes in previous commit (Change to user help.)? ^__^`

@morevnaproject  I did it by mistake :sweat_smile: 

`Ah, great! Now everything is alright! Can you please rebase your branch on top of the latest master? ^__^`

Did it! :D

File: synfig-studio/src/player/visualizationwindow.cpp [hunk 2]
```diff
@@ -251,7 +254,16 @@ VisualizationWindow::on_content_draw(const Cairo::RefPtr<Cairo::Context> &contex
 	}
 	
 	++rendered_frames;
+<<<<<<< HEAD
 	frame = (frame + 1) % frames_count;
+=======
```
Comment (ice0) [inline synfig-studio/src/player/visualizationwindow.cpp:259]: Looks like unmerged code,

**Comment by Aa20475** [comment]
`OSError: Error reading file '/home/travis/build/synfig/synfig/synfig-tests/export/lottie/*.sif': failed to load external entity "/home/travis/build/synfig/synfig/synfig-tests/export/lottie/*.sif"`

The other builds failed coz of the merge conflict flags but what about this? :thinking: :frowning_face:

**Comment by morevnaproject** [comment]
> I think is better to "Squash and merge" - also it automatically rebased.

@ice0 You right. ^__^

**Comment by morevnaproject** [comment]
Looks good now. Thank you! ^__^

**Pull request merged**